dockerfiles/cpp-23.Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM gcc:13.2.0-bookworm
+
+ENV CODECRAFTERS_DEPENDENCY_FILE_PATHS="vcpkg.json,vcpkg-configuration.json"
+
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y zip=3.* && \ 
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN pwd && \
+    git clone https://github.com/microsoft/vcpkg.git && \
+    ./vcpkg/bootstrap-vcpkg.sh -disableMetrics
+
+RUN export "VCPKG_ROOT"="$(pwd)" && \
```

Comment:
I think the more canonical way to do this would be to use `ENV`, like: 

```
ENV VCPKG_ROOT=/app
```

You can use `$PATH`-style interpolation in `ENV` statements too, if I'm not wrong.